server/models/userModel.js

```diff
@@ -4,7 +4,10 @@ var schema = Schema
 
 userSchema = new schema({
     username: {type: String, required: true},
-    password: {type: String, required: true}
+    password: {type: String, required: true},
+    notes: {type: Array[String], required: true, default: []},
+    favourites: {type: Array[String], required: true, default: []},
+    preferences: {type: Array[String], required: true, default: []},
```

Comment:
Some considerations for the notes and preferences in user schema.
> 1. Favourites: an array of restaurant Ids, so `string[]` is perfect.
> 2. Preferences: an array of filter objects like `{ type: string, value: string }[]`
> 3. Notes (each user has notes for specific restaurants): so maybe it would look more like `{ restaurantId: string, note: string }[]`